docs/klaytn-contracts/dao-contracts/sdks-and-tutorials.md

```diff
@@ -3,3 +3,567 @@ title: Tutorial
+This will create a transaction that casts your vote
+
+Paste the code below in your `governance.js` to cast your vote given your  `proposaIId`
+
+
+```js
+// Cast vote
+// This will create a transaction that casts your vote
+const castVote = async () => {
+    const proposalId = “<PASTE PROPOSALID>”;
+    const support = "1";
+
+    governanceContract.send({from: keyring.address, gas: 1500000, value: 0}, 'castVote', proposalId, support)
+    .then(receipt => {
+        console.log(receipt);
+    })
+    .catch(err => {
+        console.log(err);
+    })
+}
+
+castVote()
+``` 
+
+**Output**
+
+Run the code below in your terminal to see the transaction result
+
+```js
+node scripts/governance.js
+```
+
+You have casted your vote after a successful execution of the transaction.
+
+
+### d. Execute passed Proposal <a id="Execute passed Proposal"></a>
+Once the proposal is voted on and the voting period (5 mins) reaches the deadline, the proposal is passed. You can now execute the passed proposal.
+
+To execute:
+
+1. In `execute`, provide details:
+    * In **targets**, provide the address of your KIP7 mintable token.
+    * In **values**, provide 0 for the amount of KLAY.
+    * In **calldatas**, provide the same call data that you generated earlier for the function mint(address to, uint256 amount).
+    * In **descriptionHash**, provide the hash of the description that you used to generate the proposal. To do this the easy way, go to an online [Keccak-256 generator](https://emn178.github.io/online-tools/keccak_256.html) and paste your description text, copy the result and precede it with 0x.
+
+![](/images/dao/keccak.png)
+
+In this case, the `descriptionHash` looks like this:
+
+> 0x5972eb28939b3d790b6661496564aa3a329cbfbc0dd087bf9377a66ba43a809f
+
+This will execute the passed proposal and the governance contract will mint the KIP-7 mintable tokens to the account that you provided in the call data when creating the proposal.
+
+
+```js
+// Execute Passed Proposal
+// Will execute the passed proposal on the governance contract
+const executeProposal = async () => {
+    const target = [“<PASTE GOVERNANCE MINTABLE CONTRACT ADDRESS>”] // KIP7 Mintable Token Contract Address
+    const values = ["0"] 
+    const calldata = [“<PASTE CALLDATA HERE>”]
+    const descriptionHash = “<PASTE DESCRIPTION HASH HERE>”;
+
+
+    governanceContract.send({from: keyring.address, gas: 1500000, value: 0}, 'execute', target, values, calldata, descriptionHash)
+    .then(receipt => {
+        console.log(receipt);
+    })
+    .catch(err => {
+        console.log(err);
+    })
+}
+
+executeProposal()
+```
+
+**Output**
+
+Run the code below in your terminal to see the transaction result
+
+```js
+node scripts/governance.js
+```
+
+After successfully executing this transaction, new tokens will be minted to the address specified in your calldata.
+
+![](/images/dao/tokenDisplay.png)
+
+Congratulations on performing your on-chain governance successfully.
+
+**FullCode**
```

Comment:
Instead of fullcode can we publish this code as a gist and give the url here ? 
```suggestion
**Implementation** : gist url
```

Reply:
yeah sure